src/coreclr/vm/readytoruninfo.cpp

```diff
@@ -1929,20 +1929,17 @@ bool ReadyToRun_TypeGenericInfoMap::HasConstraints(mdTypeDef input, bool *foundR
 
 bool ReadyToRun_MethodIsGenericMap::IsGeneric(mdMethodDef input, bool *foundResult) const
 {
-#ifdef DACCESS_COMPILE
-    *foundResult = false;
-    return false;
-#else
+#ifndef DACCESS_COMPILE
     uint32_t rid = RidFromToken(input);
-    if ((rid > MethodCount) || (rid == 0))
+    if ((rid <= MethodCount) && (rid != 0))
```

Comment:
Missing return and assigning value to `foundResult` when this condition is not met.

Reply:
@fanyang-mono the return and assginment are after the `#endif`.  This PR unifies the false case and the `defined(DACCESS_COMPILE)` cases